website/docs/r/jwt_auth_backend_role.html.md

```diff
@@ -70,7 +73,7 @@ The following arguments are supported:
   this value.
 
 * `bound_claims` - (Optional) If set, a map of claims/values to match against.
-  The expected value may be a single string or a list of strings.
+  A claim's value may be a single string or a comma-separated string list.
```

Comment:
With the example above, it's perfectly clear, but just reading this inline, I wasn't sure between `["red","green","blue"]` and `"red,green,blue"`. I've had a go at clarifying in the suggestion.

```suggestion
  A claim's value must be a string, which may contain one value or multiple
  comma-separated values, e.g. `"red"` or `"red,green,blue"`.
```